### DISCUSSION TITLE
Media Library Pro Support

### DISCUSSION THREAD

**Discussion by tjhunkin-inhance** [spatie/laravel-medialibrary#3508] [Q&A]
Where do I log Media Library Pro Support issues? Logged an issue on the normal repo a week ago with zero response as yet. Perhaps paying clients should have better support?

**Comment by freekmurze** [comment]
Issues around this can go to the media library pro repo: https://github.com/spatie/laravel-medialibrary-pro
You should have gotten an invite when your purchased the package.
Alternatively, you can send a mail to support@spatie.be

**Comment by truthytech** [comment]
I no longer have the invite for https://github.com/spatie/laravel-medialibrary-pro, so I can't access it.
You also don't reply to any emails.
